### DISCUSSION TITLE
Release plan

### DISCUSSION THREAD

**Discussion by vincent-pli** [danthedeckie/simpleeval#107] [Q&A]
Do we have plan to make a new release?
we hit the issue: #69 and it has been fix by #72 
but not perform new release for it, so i wonder to know the release plan see if it can catch our time-line thanks

**Comment by n1ngu** [comment]
Also #90 would be solved by the straight-forward #91

Maybe it is time to evolve simpleeval into a community project so that @danthedeckie can be saved from some packaging burden. Are there any contributors you would trust to help maintain the package?

Also, is the release process automated in any way? Is help wanted for this? Maybe a github action would come in handy!

**Comment by n1ngu** [comment]
If in need of help, the Tryton project https://www.tryton.org/ has a strong dependency on simpleeval and I can vouch for anyone in that circle as a trustworthy FOSS maintainer. Yet, github isn't their main platform so maybe ping them in https://discuss.tryton.org/

**Comment by jeanwainer** [comment]
Also looking for a new release! Need the assignment check from the latest code

**Comment by danthedeckie** [comment]
OK, So I've put out a new release.
I don't think I did it very well.  Many many apologies to everyone depending on this.
Yes, definitely need a better future plan for this.

**Comment by jeanwainer** [comment]
Awesome, thanks Dan!

**Comment by n1ngu** [comment]
Thanks for all the effort you put in this.

As you can see, you lil' experiment from 2013 has become rather significant in the python ecosystem, so don't hesitate to ask for community help!
